orchagent/neighorch.cpp

```diff
@@ -137,16 +137,24 @@ void NeighOrch::doTask(Consumer &consumer)
         }
 
         string alias = key.substr(0, found);
-        Port p;
 
-        if (!gPortsOrch->getPort(alias, p))
+        if (alias == "eth0" || alias == "lo" || alias == "docker0")
         {
             it = consumer.m_toSync.erase(it);
             continue;
         }
 
+        Port p;
+        if (!gPortsOrch->getPort(alias, p))
+        {
+            SWSS_LOG_INFO("Port %s doesn't exist", alias.c_str());
+            it++;
+            continue;
```

Comment:
ok. help to me understand this first. In this case, why we get neighbor first before we get the teamd messages? I think the team dev should be created very early, by the time we learn a neighbor on a lag, we should already have the lag device in the APP DB, right?